course_catalog/views.py

```diff
@@ -448,3 +449,21 @@ class PodcastViewSet(viewsets.ReadOnlyModelViewSet):
         Prefetch("offered_by", queryset=LearningResourceOfferor.objects.all()),
         Prefetch("topics", queryset=CourseTopic.objects.all()),
     )
+
+
+class RecentPodcastEpisodesViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for recent PodcastEpisodes
+    """
+
+    serializer_class = PodcastEpisodeSerializer
+    permission_classes = (ReadOnly & PodcastFeatureFlag,)
+
+    queryset = (
+        PodcastEpisode.objects.filter(published=True)
+        .order_by("-last_modified", "-id")
+        .prefetch_related(
+            Prefetch("offered_by", queryset=LearningResourceOfferor.objects.all()),
+            Prefetch("topics", queryset=CourseTopic.objects.all()),
+        )
+    )
```

Comment:
does this by default return a list of all the podcast episodes? just curious if we should limit it to 10 or something (obviously ignore this comment if its already in there haha)

Reply:
Hmm, I thought `DefaultPagination` was set by default but apparently it's not for these ViewSets. It sets a limit of 10 per page with a 100 items max. I'll add it to the PR but it will change the JSON returned. Alternatively, it can put the pagination links in the header but it doesn't look like we do that for other APIs.

Do we have any code in the frontend that currently handles pagination? It looks like `/api/v0/videos/` and other APIs already use the paginated format so I assume we are doing something in the frontend already to handle that.

Reply:
we have pagination I think for the contents of user lists and a few other things

Reply:
I'll return the paginated output here. It looks like the `api/v0/videos/new` API does not handle pagination, it only works with whatever is in the first page. I guess we can handle pagination later on if necessary.

Reply:
that makes sense to me - I think in the designs right now it's only showing 6 recent episodes, so just even limiting it to that would be fine